### PR TITLE
chore: add script to setup secrets for first time infrastructure setup

### PR DESCRIPTION
**Pull request by anitarua**
Adds a first-time-setup script for creating the AWS Secrets Manager secrets expected by the moderated chat infrastructure.

### CODE REVIEW

File: infrastructure/setup-secrets.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+if [ -z "$MOMENTO_API_KEY" ]
+then
+      echo "\$MOMENTO_API_KEY is empty"
+      usage
+fi
+
+# Make sure user has aws cli installed before continuing
+doesCommandExist "aws"
+
+createOrUpdateSecret "moderator/demo/secrets" "{\"momentoApiKey\":\"${MOMENTO_API_KEY}\", \"momentoSigningSecret\":\"placeholder-will-be-replaced-during-cdk-setup\"}"
```
Comment (pratik151192) [inline infrastructure/setup-secrets.sh:71]: nitpick: `moderator` is a broad word so i'd probably use like `moderator/chat/demo/secrets`. The 3rd argument to the method `createOrUpdateSecret` is `description` which we should probably add as well. 

For my knowledge, what does `placeholder-will-be-replaced-during-cdk-setup` do and when does that happen?

**Review by pratik151192 — COMMENTED** [review]
Just a minor comment/question

File: infrastructure/setup-secrets.sh (hunk 1 above)
Comment (anitarua) [reply infrastructure/setup-secrets.sh:71]: Good points, I'll update the secret name here and in the cdk stack, and I'll add a description.

In the infra cdk stack, there's a custom resource lambda that's run with each deployment. This lambda creates the "moderator" cache and the necessary webhook if they don't exist and updates the webhook secret anytime a new webhook is created.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Contains apis for the moderated chat application. There are a few apis that we a
 `GET /v1/translate/supportedLanguages` - returns the languages that the application currently supports
 `GET /v1/translate/token/<username>` - returns a short lived token that allows <username> to publish to the `chat-publish` topic
 
-In order to run these apis, there needs to be a secret stored in aws secrets manager with the path `moderator/demo/secrets`. This secret should be key value pairs in the format
+In order to run these apis, there needs to be a secret stored in aws secrets manager with the path `moderated-chat/demo/secrets`. This secret should be key value pairs in the format
```
Comment (cprice404) [inline README.md:53]: can you mention here that you created a script to use to write the secrets?

**Review by cprice404 — APPROVED** [review]

**Pull request merged**